app/assets/javascripts/base.js.erb

```diff
@@ -114,8 +114,9 @@ var instantClick
 
   function changePage(title, body, newUrl, scrollY, pop) {
     var pageContentDiv = document.getElementById("page-content");
-    if (document.getElementById("navigation-butt")) {
-      document.getElementById("navigation-butt").classList.remove('showing')
+    var memberMenuButton = document.getElementById("member-menu-button")
```

Comment:
The change here is `navigation-butt` is now `member-menu-button`. This new ID is used with other changes in this PR.

Reply:
I didn't even know this code was in there. Thanks for the update!